conda/meta.yaml

```diff
@@ -12,10 +14,8 @@ requirements:
     - setuptools
     - setuptools_scm
   run:
-    - python >=3.8
-    - python-dateutil
     - matplotlib
-  run_constrained:
+    - python>=3.8
```

Comment:
The min version is 3.9 in other places.

Reply:
3.8/3.9 is because I asked for 3.8 when answering the questions of copier, but then was having issues on CI with the env generated by `pip-compile-multi`.

I think it's because I ran `pip-compile-multi` on my machine that has 3.10, and there is some conflict with a library `contourpy` used by `matplotlib`: see [here](https://github.com/scipp/plopp/actions/runs/6961743750/job/18944021427).

So I was lazy, didn't want to start the whole copier thing from scratch, so I tried to change it manually. I obviously forgot quite a few places, meaning that I probably should not have been lazy...

Anyway, thanks for being thorough.

Reply:
It's fine to switch to 3.9 now, I think. Running copier again should be quick, you only need to change one answer.